### ISSUE TITLE
Wrong wrapping params in step name

### ISSUE THREAD

**Issue by dsmt** [serenity-bdd/serenity-core#246]
When I add second parameter to step name HTML special chars break markup while with the first parameter everything is seemed to be Ok

        petStore.when_i_add_the_pet_to_the_store(fido, "param");

```
    @Step("Print this title <{1}>")
    public void when_i_add_the_pet_to_the_store(Pet pet, String st) {
        this.pet = pet;
        int id = Math.abs(new Random().nextInt());
        this.jsonPet = "{\"id\": " + id + " , \"name\": \""
                + pet.getName() + "\", \"photoUrls\": [], \"status\": \""
                + pet.getStatus() + "\"}";

        rest().given()
                .contentType("application/json")
                .content(jsonPet)
                .post("http://petstore.swagger.io/v2/pet");

        this.pet.setId(id);
    }
```

![](http://take.ms/ZXf7J)

**Comment by wakaleo** [comment]
The problem is the "<" in the text: if you use  @Step("Print this title {1}") or  @Step("Print this title {0},{1}"), it works. I will try to find out why it doesn't like the extra markup, but I would suggest that they don't do much for the readability of the reports.

**Comment by wakaleo** [comment]
Fix released in version 1.1.32-rc.2
